{{cookiecutter.project_slug}}/requirements/local.txt

```diff
@@ -1,6 +1,6 @@
 -r ./base.txt
 
-Werkzeug==0.15.4  # https://github.com/pallets/werkzeug
+Werkzeug==0.14  # https://github.com/pallets/werkzeug
```

Comment:
Can we use 0.14.1 instead? Also, we would want to prevent pyup from updating to 0.15 and above by adding a filter:

```suggestion
Werkzeug==0.14.1  # pyup: < 0.15  # https://github.com/pallets/werkzeug
```